docfx_project/ziti/clients/linux.md

```diff
@@ -1,59 +1,147 @@
 ## Linux
 
-The linux tunneler is capable of operating in both transparent and proxy modes. In transparent mode the tunneler will
-interact with the operating system to establish intercept points. These intercept points correspond directly to the
-service definitions in place.
+### The Tunneler CLI
 
-`ziti-tunnel` supports several intercept modes, which are specified with a sub-command.
-Typically you will run `ziti-tunnel` with the _run_ sub-command, which determines
-the preferred intercept mode based on the kernel drivers that are available on the
-host.
+`ziti-edge-tunnel` is the general purpose tunneler CLI and can also run as a systemd daemon. For the best overall experience, please use the preferred tunneler `ziti-edge-tunnel` described here.
 
-    $ sudo ziti-tunnel run ziti.json
-    [   0.000]    INFO ziti/tunnel/intercept/tproxy.New: tproxy listening on 127.0.0.1:33641
-    [   0.006]    INFO ziti/tunnel/cmd/ziti-tunnel/subcmd.run: using tproxy interceptor
-    ...
+The purpose of the tunneler is to configure host access. This means all users and all processes on the host will share the same level of access. This is accomplished by configuring the OS to have an on-board OpenZiti DNS nameserver and IP routes for authorized OpenZiti Services.
 
-When started on a host that lacks __iptables__, the tproxy intercept mode initializer
-fails and `ziti-tunnel` attempts to use the tun intercept mode:
+### Installation and Upgrade
 
-    $ sudo ziti-tunnel run ziti.json
-    [   0.001]    INFO ziti/tunnel/intercept/tproxy.New: tproxy listening on 127.0.0.1:37313
-    [   0.001]    INFO ziti/tunnel/cmd/ziti-tunnel/subcmd.run: tproxy initialization failed: failed to initialize iptables handle: exec: "iptables": executable file not found in $PATH
-    [   0.009]    INFO ziti/tunnel/cmd/ziti-tunnel/subcmd.run: using tun interceptor
+[The latest release](https://github.com/openziti/ziti-tunnel-sdk-c/releases/latest/) of `ziti-edge-tunnel` is distributed as a binary executable from GitHub. The upgrade procedure is identical to the installation procedure.
+
+```bash
+# shell script illustrating the steps to install or upgrade ziti-edge-tunnel
+wget -q "https://github.com/openziti/ziti-tunnel-sdk-c/releases/latest/download/ziti-edge-tunnel-Linux_$(uname -p).zip" \
+  && unzip ./ziti-edge-tunnel-Linux_$(uname -p).zip \
+  && rm ./ziti-edge-tunnel-Linux_$(uname -p).zip \
+  && chmod -c +x ./ziti-edge-tunnel \
+  && ./ziti-edge-tunnel version
+```
+
+Optionally, you may install `ziti-edge-tunnel.service` as a systemd unit. For example,

```

Comment:
I really try not to document HOW to use systems in this way. It's helpful, but it opens us up to some questions from the doc. we do make systems files for the quickstart in the ziti-cli-scripts.sh, maybe this is a function to just add in there and then turn this section into a "source this file and run createSystemdUnitFileForTunneller" (or whatever?)